docs/01_nodeos/02_usage/03_development-environment/10_local-single-node-testnet-consensus.md

```diff
@@ -41,30 +41,28 @@ cleos wallet import --private-key 5KQwrPbwdL6PhXujxW37FSSQZ1JiwsST4cqQzDeyXtP79z
 Start your own single-node blockchain with this single command:
 
 ```sh
-nodeos -e -p eosio --plugin eosio::chain_api_plugin --plugin eosio::history_api_plugin
+nodeos -e -p eosio --plugin eosio::producer_plugin --plugin eosio::producer_api_plugin --plugin eosio::http_plugin --plugin eosio::chain_api_plugin --filter-on="*" --access-control-allow-origin='*' --contracts-console --http-validate-host=false --verbose-http-errors
 ```
```

Comment:
Remove `--filter-on="*"` as that is for the history_plugin.
The other options I guess are ok, but why are you specifying them? None of them are needed. Seems a bit odd to specify just these random few.
All that is needed is `--plugin eosio::chain_api_plugin` to enable interaction via `cleos`.